x/incentives/keeper/distribute.go

```diff
@@ -267,59 +383,6 @@ func (k Keeper) distributeSyntheticInternal(
 	return k.distributeInternal(ctx, gauge, sortedAndTrimmedQualifiedLocks, distrInfo)
 }
 
-// AllocateAcrossGauges gets all the active groupGauges and distributes tokens based on the splitting policy of each gauge.
-// After each iteration we update the groupGauge by modifying filledEpoch and distributed coins.
-func (k Keeper) AllocateAcrossGauges(ctx sdk.Context) error {
```

Comment:
Note to reviewer: I purposefully moved the method to the top to avoid unclear diff since I changed more than originally expected